banking/nlu.md

```diff
@@ -11,18 +11,31 @@
 - South Korean Won
 - Swiss Franc
 - United States Dollar
+- Pound Sterling
+- Canadian Dollar
+- Hong Kong Dollar
+- Swedish Krona
+- Singapore Dollar
 
 # intent:transfer_money <!--The user would like to transfer money.-->
 
 - I would like to make a transfer
 - I want to transfer money
+- I want to pay money
+- I want to pay someone
+- Can I transfer money?
+- pay my bills
```

Comment:
I think "pay bills" would probably be a different intent, usually it would lead to something else

Reply:
I have removed that line. Now that I think about it, it should have been a different intent